### ISSUE TITLE
[TalkBack] Talkback Explore by touch off -> reboot-> Explore by touch on

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#230]
```
What steps will reproduce the problem?
Talkback on -> Setting-> Explore by touch off -> reboot ->  Explore by touch On


What is the expected output? What do you see instead?
Explore by touch off

What version of the product are you using? On what operating system?
android 4.2.1

Please provide any additional information below.

```

Original issue reported on code.google.com by `ezc...@gmail.com` on 18 Apr 2013 at 11:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
Cannot reproduce using latest beta version of TalkBack (v3.3.2_b1).
```

Original comment by `alanv@google.com` on 22 Apr 2013 at 7:02
- Changed state: **WontFix**
